core/meterer/dynamodb_metering_store_test.go

```diff
@@ -72,46 +72,74 @@ func cleanupTables(tc *testContext) {
 	_ = dynamoClient.DeleteTable(tc.ctx, tc.globalBinTable)
 }
 
-// TestUpdateReservationBin tests the UpdateReservationBin function
-func TestUpdateReservationBin(t *testing.T) {
-	tc := setupTest(t)
-
-	// Test updating bin that doesn't exist yet (should create it)
-	accountID := gethcommon.HexToAddress("0x1234567890123456789012345678901234567890")
-	reservationPeriod := uint64(1)
-	size := uint64(1000)
+// TestIncrementBinUsages_EdgeCases tests the IncrementBinUsages function with edge cases
+func TestIncrementBinUsages_EdgeCases(t *testing.T) {
+	t.Run("empty input", func(t *testing.T) {
+		tc := setupTest(t)
+		binUsages, errs := tc.store.IncrementBinUsages(tc.ctx, gethcommon.HexToAddress("0xabcdefabcdefabcdefabcdefabcdefabcdefabcd"), []core.QuorumID{}, map[core.QuorumID]uint64{}, map[core.QuorumID]uint64{})
+		assert.Empty(t, binUsages)
+		assert.Empty(t, errs)
+	})
 
-	binUsage, err := tc.store.UpdateReservationBin(tc.ctx, accountID, reservationPeriod, size)
-	require.NoError(t, err)
-	assert.Equal(t, size, binUsage)
+	t.Run("exceed transaction limit", func(t *testing.T) {
+		tc := setupTest(t)
+		accountID := gethcommon.HexToAddress("0xabcdefabcdefabcdefabcdefabcdefabcdefabcd")
+		reservationPeriod := uint64(42)
+		size := uint64(100)
+		var quorums []core.QuorumID
+		periods := make(map[core.QuorumID]uint64)
+		for i := 0; i < 26; i++ {
```

Comment:
This seems like a DynamoDB specific quirk. Is there a protocol limit on the number of quorums we could use instead?